src/app/builder/builder.component.html

```diff
@@ -25,6 +25,16 @@
       width="600"
       [src]="previewIframeSrc"></iframe>
   </div>
+  <div *ngIf="playPlaylist" id="preview-playlist-wrap">
```

Comment:
OK, I can get this to happen when I check 'Make it a playlist' and then after that, choose a specific episode to tie it to. Then the preview goes back to being a single ep preview. Will fix!